decision_making.md

```diff
@@ -0,0 +1,42 @@
+# Decision Making Guide
+
+This document describes how Jupyter’s governing bodies (Board of Directors, Software Steering Council, Software Projects, non-software Working Groups) make decisions. Because individuals often work across multiple decision making groups, all Jupyter governing bodies are expected to follow this approach.
+
+This document provides guidelines but is not meant to be a comprehensive procedure manual. We seek to honor the principles of collaboration, inclusive participation, and responsive decision making. If a situation arises that is not explicitly covered by the guidelines in this document, we encourage all teams to find a path forward with decisions, filling in the gaps in formal decision making procedure with adherence to these principles.
```

Comment:
I find this sentence a bit confusing. When I first read it, I interpreted the rest of the page's content as "these are suggestions but not rules". But then in the next section it says things like `"Formally, these decision making bodies make decisions using consensus seeking with an option to call a vote to move the decision forward."` which seems like something closer to a mandate. Which interpretation is correct?